guardrails/llm_providers.py

```diff
@@ -498,6 +498,10 @@ def _invoke_llm(
             ),
         )
 
+        # these are gr only and should not be getting passed to llms
```

Comment:
if you set reasks to 0 from the server these flow through as none 🤔 i think we can drop this in 060 i dont think my other branch supports it